cookieMaxAgeInMonths to migration guide (#270)
front-commerce/developers.front-commerce.com: source/docs/appendices/migration-guides.md

## `2.11.0` -> `2.12.0`

> ### cookies max age config

Reviewer: ```suggestion
### Cookies max age configuration
```

> `cookieMaxAgeInMonths` configuration in `src/config/website.js` which represents the consent cookie's maxage in months now has a default value of 12 months. Previously if left unconfigured the cookie banner will appear to the user everytime he revisits the site.

Reviewer: ```suggestion
The `cookieMaxAgeInMonths` configuration in `src/config/website.js` represents the consent cookie's maxage in months . It now has a default value of **12 months**. Previously if left unconfigured the cookie banner will appear to users every time they visit the site.
```
